### PR TITLE
Update content security policy rules

### PR DESCRIPTION
**Pull request by san650**
Allow omdb, media-imdb and firebase sources

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -16,6 +16,16 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+
+    contentSecurityPolicy: {
+      'default-src': "'none'",
+      'script-src': "'self' 'unsafe-eval'",
```
Comment (eldano) [inline config/environment.js]: I had to add a 3rd option to the 'script-src' policy to get rid of all errors:
'script-src': "'self' 'unsafe-eval' https://cdn.firebase.com",

Comment (san650) [inline config/environment.js]: Please, commit the change and update this branch.

**Comment by san650** [comment]
Fixes #2

**Comment by eldano** [comment]
Thanks!

**Pull request merged**